src/components/Addtransaction.js

```diff
@@ -0,0 +1,92 @@
+import { Button, Modal, InputGroup, Form, FormControl } from 'react-bootstrap';
+import React from 'react';
+import { useState } from 'react';
+import Senddata from './Senddata';
+
+
+function Addtransaction(props) {
+    const options = [
+        {
+            key: 1,
+            label: "Choose Transaction Type",
+            value: "-",
+        },
+        {
+            key: 2,
+            label: "Debit",
+            value: "debit",
+        },
+        {
+            key: 3,
+            label: "Credit",
+            value: "credit",
+        }];
+    const [transactionType, setTransactionType] = useState();
+    const [transactionAmount, setTransactionAmount] = useState();
+    const [transactionDescription, setTransactionDescription] = useState();
```

Comment:
Always initialise the state with the intial value for that state.

Like for transactionAmount  it should be 0.
transactionDescription it should be  ''.
accordingly a default transactionType.

Reply:
Done sir.